.github/workflows/main.yaml

```diff
@@ -21,4 +21,4 @@ jobs:
       - name: install
         run: npm install
       - name: build
-        run: npx roamjs-scripts build --labs
+        run: npx samepage build
```

Comment:
I'd like to start slowly phasing out `roamjs-scripts` in favor of `samepage`, which will come bundled with `roamjs-components` as part of `@samepage/scripts`